person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require './nameable'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(age, name: 'Unknown', parent_permission: true)
+    super()
     @id = SecureRandom.uuid
```

Comment:
- [ ] **[OPTIONAL]** I had issues testing your app due to this `SecureRandom` module you are using here. It needs to be imported at the top of the class.

  ```
  require 'securerandom'
  ```

  ![image](https://github.com/Ayokunnumi1/school-library/assets/36542986/91576e3f-34da-4ed1-abd9-7efd1a5c7faa)

Reply:
Thanks for your corrections, I have resolved them